queries/c/locals.scm

```diff
@@ -8,8 +8,12 @@
   name: (identifier) @definition.macro)
 (pointer_declarator
   declarator: (identifier) @definition.var)
-(parameter_declaration
-  declarator: (identifier) @definition.parameter)
+; (parameter_declaration
+;   declarator: (identifier) @definition.parameter)
+
+((identifier) @definition.parameter
+  (#has-ancestor? @definition.parameter "parameter_declaration"))
+
```

Comment:
I guess removing is sufficient since this query already comes from C

Reply:
Wornf position. I mean in the copy file. I'm on mobile right now. Too Little screen to big fingers...